src/main/resources/db/demo-data/buq.source_of_fund.csv

```diff
@@ -0,0 +1,4 @@
+id,name,description
+08b2b579-8e08-45f2-a640-5f558713b374,Health Basket Fund,Source of fund description 1
+4f441cfc-0556-46f5-b2ca-ad7988d8781a,Donor funding,Source of fund description 2
+bb790da3-d1cc-4b33-af34-75abdb17e50f,Other insurance schemes,Source of fund description 3
```

Comment:
I assume this should be deleted

Reply:
I resolved conflicts.
Why would we delete it? This is demo data (like [this](https://github.com/OpenLMIS/openlmis-requisition/blob/master/src/main/resources/db/demo-data/requisition.requisitions.csv)). This will save us from manually adding them for development, testing, etc.

Reply:
@sradziszewski I'm sorry, I didn't realize it is put in the `demo-data`